### PR TITLE
Fixes #1672 | Adds support for TouchEvents on ClayTooltip

### LINKED ISSUE #1672: ClayTooltip doesn't supports touch events
Try run the demo on a mobile phone

### PR DESCRIPTION
**Pull request by diegonvs**
As decided in https://docs.google.com/document/d/1uYT_r59q-GisQEHuRmucFCkzKnmILFY2vil8WgEtQhg/edit?ts=5c90adf1 :
> On touch interfaces tooltip have to appear on long press. The delay time for it is 500ms.

### CODE REVIEW

File: packages/clay-tooltip/src/ClayTooltip.js [hunk 1]
```diff
@@ -187,7 +188,48 @@ class ClayTooltip extends ClayComponent {
...
 	/**
-	 * Handles transionend event.
+	 * Handles touchstart events.
+	 * @memberOf ClayTooltip
+	 * @param {!Event} event
+	 * @private
+	 */
+	_handleTouchStart(event) {
+		this._timer = setTimeout(() => {
+			const content = this._getContent(event.srcElement);
```
Comment (carloslancha) [inline packages/clay-tooltip/src/ClayTooltip.js]: This, but the last line, is the same code as present in `_handleMouseEnter`. You can extract it to a method `_showTooltip`

File: packages/clay-tooltip/src/ClayTooltip.js [hunk 2]
```diff
@@ -187,7 +188,48 @@ class ClayTooltip extends ClayComponent {
...
+	 * @memberOf ClayTooltip
+	 * @param {!Event} event
+	 * @private
+	 */
+	_handleTouchEnd(event) {
+		if (this._timer) {
+			clearTimeout(this._timer);
+			this._timer = undefined;
+		}
+		if (this.visible) {
```
Comment (carloslancha) [inline packages/clay-tooltip/src/ClayTooltip.js]: This is the same code as present in _handleMouseLeave. You can extract it to a method _hideTooltip

Comment (diegonvs) [reply packages/clay-tooltip/src/ClayTooltip.js]: makes sense @carloslancha!

File: packages/clay-tooltip/src/ClayTooltip.js (hunk 1 above)
Comment (diegonvs) [reply packages/clay-tooltip/src/ClayTooltip.js]: Done

**Comment by carloslancha** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by carloslancha** [comment]
Merged, thx!

**Pull request merged**